--- Makefile.in.orig	2007-04-14 00:53:07.000000000 +0400
+++ Makefile.in	2007-04-16 10:18:16.000000000 +0400
@@ -230,6 +230,12 @@
 mod_proxy_la_LINK = $(LIBTOOL) --tag=CC $(AM_LIBTOOLFLAGS) \
 	$(LIBTOOLFLAGS) --mode=link $(CCLD) $(AM_CFLAGS) $(CFLAGS) \
 	$(mod_proxy_la_LDFLAGS) $(LDFLAGS) -o $@
+mod_lisp_la_DEPENDENCIES = $(am__DEPENDENCIES_2)
+am_mod_lisp_la_OBJECTS = mod_lisp.lo
+mod_lisp_la_OBJECTS = $(am_mod_lisp_la_OBJECTS)
+mod_lisp_la_LINK = $(LIBTOOL) --tag=CC $(AM_LIBTOOLFLAGS) \
+	$(LIBTOOLFLAGS) --mode=link $(CCLD) $(AM_CFLAGS) $(CFLAGS) \
+	$(mod_lisp_la_LDFLAGS) $(LDFLAGS) -o $@
 mod_redirect_la_DEPENDENCIES = $(am__DEPENDENCIES_1) \
 	$(am__DEPENDENCIES_2)
 am_mod_redirect_la_OBJECTS = mod_redirect.lo
@@ -401,7 +407,7 @@
 	$(mod_extforward_la_SOURCES) $(mod_fastcgi_la_SOURCES) \
 	$(mod_flv_streaming_la_SOURCES) $(mod_indexfile_la_SOURCES) \
 	$(mod_magnet_la_SOURCES) $(mod_mysql_vhost_la_SOURCES) \
-	$(mod_proxy_la_SOURCES) $(mod_redirect_la_SOURCES) \
+	$(mod_proxy_la_SOURCES) $(mod_lisp_la_SOURCES) $(mod_redirect_la_SOURCES) \
 	$(mod_rewrite_la_SOURCES) $(mod_rrdtool_la_SOURCES) \
 	$(mod_scgi_la_SOURCES) $(mod_secdownload_la_SOURCES) \
 	$(mod_setenv_la_SOURCES) $(mod_simple_vhost_la_SOURCES) \
@@ -419,7 +425,7 @@
 	$(mod_expire_la_SOURCES) $(mod_extforward_la_SOURCES) \
 	$(mod_fastcgi_la_SOURCES) $(mod_flv_streaming_la_SOURCES) \
 	$(mod_indexfile_la_SOURCES) $(mod_magnet_la_SOURCES) \
-	$(mod_mysql_vhost_la_SOURCES) $(mod_proxy_la_SOURCES) \
+	$(mod_mysql_vhost_la_SOURCES) $(mod_proxy_la_SOURCES) $(mod_lisp_la_SOURCES) \
 	$(mod_redirect_la_SOURCES) $(mod_rewrite_la_SOURCES) \
 	$(mod_rrdtool_la_SOURCES) $(mod_scgi_la_SOURCES) \
 	$(mod_secdownload_la_SOURCES) $(mod_setenv_la_SOURCES) \
@@ -608,7 +614,7 @@
 	mod_ssi.la mod_secdownload.la mod_expire.la mod_evhost.la \
 	mod_simple_vhost.la mod_fastcgi.la mod_extforward.la \
 	mod_access.la mod_compress.la mod_auth.la mod_rewrite.la \
-	mod_redirect.la mod_status.la mod_accesslog.la
+	mod_redirect.la mod_status.la mod_accesslog.la mod_lisp.la
 @NO_RDYNAMIC_TRUE@liblightcomp_la_SOURCES = $(common_src)
 @NO_RDYNAMIC_TRUE@liblightcomp_la_CFLAGS = $(AM_CFLAGS)
 @NO_RDYNAMIC_TRUE@liblightcomp_la_LDFLAGS = -avoid-version -no-undefined
@@ -673,6 +679,9 @@
 mod_proxy_la_SOURCES = mod_proxy.c
 mod_proxy_la_LDFLAGS = -module -export-dynamic -avoid-version -no-undefined
 mod_proxy_la_LIBADD = $(common_libadd)
+mod_lisp_la_SOURCES = mod_lisp.c
+mod_lisp_la_LDFLAGS = -module -export-dynamic -avoid-version -no-undefined
+mod_lisp_la_LIBADD = $(common_libadd)
 mod_ssi_la_SOURCES = mod_ssi_exprparser.c mod_ssi_expr.c mod_ssi.c
 mod_ssi_la_LDFLAGS = -module -export-dynamic -avoid-version -no-undefined
 mod_ssi_la_LIBADD = $(common_libadd) $(PCRE_LIB)
@@ -848,6 +857,8 @@
 	$(mod_mysql_vhost_la_LINK) -rpath $(libdir) $(mod_mysql_vhost_la_OBJECTS) $(mod_mysql_vhost_la_LIBADD) $(LIBS)
 mod_proxy.la: $(mod_proxy_la_OBJECTS) $(mod_proxy_la_DEPENDENCIES) 
 	$(mod_proxy_la_LINK) -rpath $(libdir) $(mod_proxy_la_OBJECTS) $(mod_proxy_la_LIBADD) $(LIBS)
+mod_lisp.la: $(mod_lisp_la_OBJECTS) $(mod_lisp_la_DEPENDENCIES) 
+	$(mod_lisp_la_LINK) -rpath $(libdir) $(mod_lisp_la_OBJECTS) $(mod_lisp_la_LIBADD) $(LIBS)
 mod_redirect.la: $(mod_redirect_la_OBJECTS) $(mod_redirect_la_DEPENDENCIES) 
 	$(mod_redirect_la_LINK) -rpath $(libdir) $(mod_redirect_la_OBJECTS) $(mod_redirect_la_LIBADD) $(LIBS)
 mod_rewrite.la: $(mod_rewrite_la_OBJECTS) $(mod_rewrite_la_DEPENDENCIES) 
@@ -1052,6 +1063,7 @@
 @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/mod_magnet_la-mod_magnet_cache.Plo@am__quote@
 @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/mod_mysql_vhost_la-mod_mysql_vhost.Plo@am__quote@
 @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/mod_proxy.Plo@am__quote@
+@AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/mod_lisp.Plo@am__quote@
 @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/mod_redirect.Plo@am__quote@
 @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/mod_rewrite.Plo@am__quote@
 @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/mod_rrdtool.Plo@am__quote@
